### DISCUSSION TITLE
Hi! Quick question

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [gregsullivan/_tw#48] [Q&A] [answered]
Hello Greg! 
Amazing way to create themes with tailwind! 
Im really new into it with all npm stuff and so on, basically was working with bootstrap only but have few questions.

- Can i integrate daisyui into it?
Once its shipped into compiled theme, thos blocks become dynamic? i mean my theme components will work fine with thos elements? 

If we took https://bootscore.me/ can i rebuild it completely using tailwind? And approaching it like author did with child theme?

As a guy added Vue to it, can we add react? 

I hope my questions are pretty clear, since my dev approach was a little bit different at this point. 

Ty so much, great stuff.

**Comment by gregsullivan** [comment, marked as answer]
Hi Richard—thanks for posting your questions!

I've never worked with daisyUI, but I can't see why it wouldn't work. I'm not sure what you mean about blocks becoming dynamic, though. I've used _tw with Tailwind UI, and I've heard from people using it with other component libraries, for whatever that's worth.

I'm not sure how bootScore is structured, but I suspect you may face an uphill battle trying to convert an opinionated theme using a different CSS library to use Tailwind instead. That said, I would also be surprised if it were impossible.

Truthfully, I would probably only use Vue or React with WordPress to create a headless site (excluding React in the admin area), so I'd need to hear more about your use case. How would you be integrating React?

**Comment by ritchiemarkov** [comment]
Hello Greg. Sorry about unclear question. I have managed to answer all my thought by myself. Actually your theme is made upon _s, so as well as bootscore. 
I have to deep dive into development, and my sincere congratulations on this wonderful way of theme generation. 
Said that be ready for couple of my low experience questions :D

**Reply by gregsullivan** [reply]
No problem at all—very happy to hear you were able to sort things out yourself!
